app/routes/monthlychallenges/july-2022.jsx

```diff
@@ -0,0 +1,176 @@
+import { json } from '@remix-run/node';
+import { createMetaData } from '~/util/createMetaData.server';
+
+export const handle = {
+	listTitle: 'July, 2022: Demo in public!',
+	meta: {
+		title: 'Monthly Theme & Challenge for July, 2022: Build in Public',
+		description:
+			"July challenge -> Let's build in public, starting with Daily Standup! During this month, let's learn to communicate what we're working on, show our development, be confident and proud of any progress we've made, and demo what we've done!",
+	},
+	date: '2022-07-01',
+	hero: {
+		heroHeader: '',
+	},
+};
+
+export async function loader() {
+	const { title, description } = handle.meta;
+	return json({
+		meta: createMetaData({ title, description }),
+	});
+}
+
+export function meta({ data: { meta } = {} } = {}) {
+	return meta;
+}
+
+export default function Challenge() {
+	return (
+		<>
+			<div className="alert alert-success">
```

Comment:
This div with the alert-success needs to be removed. We add these when a challenge is completed, so it was probably there from the last challenge.